superset/bin/superset

```diff
@@ -4,7 +4,61 @@ from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
 
-from superset.cli import manager
+import argparse
+import sys
+
+import gunicorn.app.base
+
+
+class GunicornSupersetApplication(gunicorn.app.base.BaseApplication):
```

Comment:
I tried that but got lots of DB errors due to forking problems. :-( The issue is that, with the way things are set up now, you can't import `superset` or any of its subpackages before gunicorn forks (i.e., in the `load` function). The underlying cause is that sqlalchemy (initialized in `superset/__init__.py`) is not fork-safe.

We ran into similar issues when pex-ifying knowledge repo. The way around it is to refactor the code so that `cli.py` can be loaded without initializing the flask app and its associated DB connections. Looking at the code, this is doable, but would require a significant amount of refactoring.

Given these limitations, can we keep the gunicorn wrapper in the `superset` script for now and leave the refactoring as a future TODO?